orcid-persistence/src/test/java/org/orcid/persistence/dao/AddressDaoTest.java

```diff
@@ -1,124 +0,0 @@
-/**
```

Comment:
Hey @rcpeters why you deleted this class? AddressDaoTest.java ... AddressDao should not be affected by this pull request, right?